src/js/angular/clustermanagement/controllers.js

```diff
@@ -312,19 +285,19 @@ const getAdvancedOptionsClass = function () {
 };
 
 CreateClusterCtrl.$inject = ['$scope', '$modalInstance', '$timeout', 'ClusterRestService', 'toastr', '$translate', 'data', '$modal',
-    'LocationsRestService'];
+    'RemoteLocationsService'];
 
-function CreateClusterCtrl($scope, $modalInstance, $timeout, ClusterRestService, toastr, $translate, data, $modal, LocationsRestService) {
+function CreateClusterCtrl($scope, $modalInstance, $timeout, ClusterRestService, toastr, $translate, data, $modal, RemoteLocationsService) {
     $scope.pageTitle = $translate.instant('cluster_management.cluster_page.create_page_title');
     $scope.autofocusId = 'autofocus';
     $scope.errors = [];
     $scope.clusterConfiguration = {
```

Comment:
Didn't Pavel mentioned that these default configs should come from the backend?!

Reply:
Yes, and there is an issue in discussion. Until the backend endpoint is available, we keep it like this.